Jenkinsfile

```diff
@@ -54,14 +54,18 @@ def smokeTest(gitUtils, bazel, utils) {
       def prUrl = utils.failIfNullOrEmpty(env.GITHUB_PR_URL)
       def repo = prUrl.split('/')[4]
       def hub = 'gcr.io/istio-testing'
+      def logHost = 'stackdriver'
+      def projID = 'istio-testing'
       switch (repo) {
         case 'pilot':
           def istioctlUrl = "https://storage.googleapis.com/istio-artifacts/${repo}/${prSha}/artifacts/istioctl"
           kubeTestArgs = "-m ${hub},${prSha} " +
               "-i ${istioctlUrl}"
           e2eArgs += "--pilot_hub=${hub}  " +
               "--pilot_tag=${prSha} " +
-              "--istioctl_url=${istioctlUrl}"
+              "--istioctl_url=${istioctlUrl} " +
+              "--log_provider=${logHost} " +
+              "--project_id=${projID}"
```

Comment:
Do you have a run when env.PROJECT did not work. There is no reason why it should not.